### PR TITLE
[Task #876] Add license scanning setup to the template

### PR DESCRIPTION
**Pull request by anamarijabalaban**
## Summary

Introduce a license scanning to the Rails template.

**Related issue**:  None

### Type

- [x] **Feature**: This pull request introduces a new feature.
- [ ] **Bug fix**: This pull request fixes a bug.
- [ ] **Refactor**: This pull request refactors existing code.
- [ ] **Documentation**: This pull request updates documentation.
- [ ] **Other**: This pull request makes other changes.

#### Additional information

- [ ] This pull request introduces a **breaking change**.

### Description
Add license_finder recipe and run license_finder commands in run_generators.rb.

## Checklist

- [x] I have performed a self-review of my own code.
- [x] I have tested my changes, including edge cases.
- [ ] I have added necessary tests for the changes introduced (if applicable).
- [ ] I have updated the documentation to reflect my changes (if applicable).

### CODE REVIEW

File: recipes/license_finder.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  <<~HEREDOC
+
+    LicenseFinder:
+      enabled: true
+      on_warn: fail
+      command: ['bundle', 'exec', 'license_finder']
+      requires_files: true
+      include:
+        - 'Gemfile*'
+        - 'package.json'
```
Comment (vr4b4c) [inline recipes/license_finder.rb:19]: I think this needs to be indented one more level to the left to be correctly nested under `PreCommit` section.

**Review by vr4b4c — CHANGES_REQUESTED** [review]

**Review by vr4b4c — APPROVED** [review]
